src/Repository/User/UserRepository.php

```diff
@@ -5,14 +5,19 @@
 namespace App\Repository\User;
 
 use App\Entity\User;
+use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bridge\Doctrine\RegistryInterface;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
 
 class UserRepository extends ServiceEntityRepository implements UserRepositoryInterface
 {
-    public function __construct(RegistryInterface $registry)
+    private $entityManager;
+
+    public function __construct(RegistryInterface $registry, EntityManagerInterface $entityManager)
```

Comment:
You already have the EntityManager here, using `getEntityManager()` (from `ServiceEntityRepository` which is an [`EntityRepository`](https://www.doctrine-project.org/api/orm/latest/Doctrine/ORM/EntityRepository.html)